### PR TITLE
font-spleen: include otb version

### PR DESCRIPTION
**Pull request by Barbaross93**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/font-spleen/template [hunk 1]
```diff
@@ -42,5 +42,8 @@ do_install() {
 	done
 	vinstall fonts.alias 644 usr/share/fonts/X11/spleen
 
+	for f in *.otb; do
```
Comment (classabbyamp) [inline srcpkgs/font-spleen/template:45]: could the loop in line 40 be turned into `for f in *.pcf.gz *.otb; do`?

**Comment by paper42** [comment]
please add your commit email to your github profile

File: srcpkgs/font-spleen/template (hunk 1 above)
Comment (Barbaross93) [reply srcpkgs/font-spleen/template:45]: Sure! I wasn't sure if it made sense to put the otb versions in the same directory, but I haven't had any problems doing that so your suggestion sounds good to me.

**Comment by Barbaross93** [comment]
@paper42 I think I did what you asked? I'm not sure though. I merely made my email public.

**Pull request merged**